platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1456,28 +1452,33 @@ public LearnerTreeView<Long> buildLearnerView(final ReconnectConfig reconnectCon
         final VirtualStateAccessor originalState = originalMap.getState();
         nodeRemover = new ReconnectNodeRemover<>(
                 originalMap.getRecords(), originalState.getFirstLeafPath(), originalState.getLastLeafPath());
-        final LearnerTreeView<Long> learnerTreeView;
-        switch (config.reconnectMode()) {
-            case "pullTopToBottom":
+        return switch (config.reconnectMode()) {
+            case "push" -> new LearnerPushVirtualTreeView<>(
```

Comment:
Same as above: it's best to use those constants that you defined in the config class.

Reply:
OK